Sources/DangerFixtures/DangerDSLResources/DSLGitLabProjectMilestone.swift

```diff
@@ -0,0 +1,18 @@
+// swiftlint:disable line_length
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'l...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerFixtures/DangerDSLResources/DSLGitLabProjectMilestone.swift;
  Line: 1;
-->

- :warning: SwiftLint rule 'line_length' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)